src/lsptoolshost/razorCommands.ts

```diff
@@ -133,8 +133,8 @@ export function registerRazorCommands(context: vscode.ExtensionContext, language
     // a project. We want to defer this work until necessary, so this command is called by the Razor document manager to tell
     // us when they need us to initialize the Razor things.
     context.subscriptions.push(
-        vscode.commands.registerCommand(razorInitializeCommand, async () => {
-            await languageServer.sendNotification('razor/initialize', {});
+        vscode.commands.registerCommand(razorInitializeCommand, async (pipeName) => {
+            await languageServer.sendNotification('razor/initialize', { pipeName: pipeName });
```

Comment:
Since the parameters object shape has to match what's in Roslyn, should there be a proper params type created? ie, a RazorInitializeParams equivalent.

At the very least a comment with a link to the class in Roslyn maybe?